paper-badge.html

```diff
@@ -181,7 +182,7 @@
 
       _updateTarget: function() {
         this._target = this.target;
-        this.async(this.notifyResize, 1);
+        requestAnimationFrame(this.notifyResize.bind(this));
```

Comment:
Well it will happen every time you update the `target` (which we just happen to not do a lot). I would change it yeah :( (bind is slow)

Sorry to be a pain! <3